programs/x509/cert_write.c

```diff
@@ -791,7 +827,10 @@ int main( int argc, char *argv[] )
 exit:
     mbedtls_x509write_crt_free( &crt );
     mbedtls_pk_free( &loaded_subject_key );
-    mbedtls_pk_free( &loaded_issuer_key );
+    if ( is_remote_key( opt.issuer_key ) )
+        mbedtls_pk_remote_free( &loaded_issuer_key );
```

Comment:
The function `mbedtls_pk_remote_free` should not exist. Instead, call `mbedtls_pk_free` normally, and implement the `free_func` method in the `pk_info_t` structure.